docs/README.md

```diff
@@ -1848,14 +1837,17 @@ METHOD: "POST"
 }
 ```
 
+>**Note**: Remember that, in case you want to query a read model that is restricted to a specific set of roles, you must send the **access token** retrieved upon sign-in. Check ["Authorizing operations"](#authorizing-operations) to know how to do this.
+
 #### Subscribing to read models
 
-To subscribe to a specific read model, we need to use a "subscription" operation, and it must be _sent through the **websocketURL**_.
+To subscribe to a specific read model, we need to use a subscription operation, and it must be _sent through the **websocketURL**_ using the [_GraphQL over WebSocket_ protocol](#the-graphql-over-websocket-protocol).
 
+Doing this process manually is a bit cumbersome. _You will probably never need to do this_, as GraphQL clients like [Apollo](#using-apollo-client) abstract this process away. However, we will explain how to do it for learning purposes. 
```

Comment:
Definitely an interesting article that we could publish, but it could make sense to keep a copy in the "going deeper" guide. I think it's worth having an easy to find explanation at hand in case someone wants to implement the protocol themselves...